06_create_cluster.sh

```diff
@@ -36,4 +36,20 @@ if [[ ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then
       --type merge --patch "{\"spec\":{\"samplesRegistry\":\"${LOCAL_REGISTRY_DNS_NAME}\"}}"
 fi
 
+if [[ ! -z "${APPLY_EXTRA_WORKERS}" ]]; then
+    if [[ ${NUM_EXTRA_WORKERS} -ne 0 && -f "${OCP_DIR}/extra_host_manifests.yaml" ]]; then
+        oc apply -f "${OCP_DIR}/extra_host_manifests.yaml"
+
+        for h in $(jq -r '.[].name' ${EXTRA_BAREMETALHOSTS_FILE}); do
+            while ! oc get baremetalhost -n openshift-machine-api $h 2>/dev/null; do
+                echo "Waiting for $h"
+                sleep 5
```

Comment:
sleep is fine at the beginning but  checking the status of the host would be better

Reply:
We discussed checking the status, but what is the status we wait seems to be a intricate.

Reply:
alright, I guess it's fine for now